event_app/components/django.py

```diff
@@ -0,0 +1,42 @@
+"""
+Django settings for EroticArtsX project.
+
+Generated by 'django-admin startproject' using Django 3.1.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+from .common import *
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = env('SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = env('DEBUG', default=False)
+
+ALLOWED_HOSTS = env('ALLOWED_HOSTS', default=['localhost', '127.0.0.1', 'app'])
+
+ROOT_URLCONF = 'event_app.urls'
+
+WSGI_APPLICATION = "event_app.wsgi.application"
+
+SITE_NAME = env("SITE_NAME", default="TEDx App")
+ADMIN_SITE_HEADER = env("ADMIN_SITE_HEADER", default="TEDx App Admin")
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/3.1/howto/static-files/
+
+STATIC_URL = '/static/'
+STATIC_ROOT = BASE_DIR / 'static'
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+)
+
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
Is it useful? because static files are being served through Nginx.

Reply:
It's useful in local environments when the Gunicorn port is exposed directly.
And also because I use Cloudflare in my own projects, the static files get served by the CDN itself. More on that here: http://whitenoise.evans.io/en/stable/#infrequently-asked-questions